portfolio/src/main/java/com/google/sps/data/Task.java

```diff
@@ -0,0 +1,39 @@
+package com.google.sps.data;
+
+/** Models a task that the user creates. */
+public final class Task {
+
+  private final String name;
+  private final String description;
+  private final Integer durationMinute;
```

Comment:
Consider using the Duration class.
https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html

Again, it might be a good idea to start with the right data types, so you don't have to refactor later.

Reply:
Done.